docs/DeveloperGuide.adoc

```diff
@@ -361,22 +367,35 @@ Windows, Linux, Unix, OS-X
 [[private-contact-detail]] Private contact detail::
 A contact detail that is not meant to be shared with others
 
+[[logging]] Logging::
+Logging is the use of logging tools to track the performance of the application while it is running.
```

Comment:
"log file

A file-based repository for information of various kinds about activity on a computer system. Administrators often consult log files; ordinary users generally do not."(from Microsoft glossary)
Seems more intuitive, what do you think? 😄